### ISSUE TITLE
Validate/explore tools - CBIBS

### ISSUE THREAD

**Issue by kbailey-noaa** [asascience-open/IOOS-EDS#150]
@Bobfrat you mentioned bugs in the validate tool before, so not sure if that was something that you're already aware of / dealing with.

A couple things:
1) In the explore tool, if I choose a CBIBS station (e.g. stingray point) 2 stations pop up (they are the same station but from 2 different sources - NDBC and CBIBS).  The station named "Station 44058" is from NDBC.  Can you remove (or hide?) the NDBC list of CBIBS stations from the explore tool? 
![explore_cbibs](https://user-images.githubusercontent.com/11823385/43590190-6cde2ac6-963e-11e8-8af8-2572ff4406ba.PNG)

2) In the validate tool, only the NDBC station shows up as an option - none of the CBIBS-sourced stations do.  
![validation_cbibs](https://user-images.githubusercontent.com/11823385/43590586-526fbe74-963f-11e8-85bd-d97f4e51a204.PNG)

This is an issue because I was trying to compare/do an analysis on water temp data and the NDBC CBIBS stations weren't returning data but the CBIBS stations were.  But beyond that I think it's confusing to users to see 2 options for the same station.

**Comment by kknee** [comment]
@Bobfrat I believe we had NDBC stations filtered out of the map before, and put them back in while CBIBS was down.  Can you filter those out again in addition to adding to explore/validate tools?

**Comment by kknee** [comment]
@Bobfrat actually, looks like you already did filter?

**Comment by Bobfrat** [comment]
Yes and the should be in the Explore Tool as well

**Comment by kbailey-noaa** [comment]
@Bobfrat Was the map feature removed?  I could've sworn there was a map in the lower right corner of the explore and/or validate tools...
I also can't find CBIBS stations anymore in the drop down menu.  In the validate tool I chose water temp, and for the station tried to find Gooses Reef and Potomac in the drop down menu but they didn't show up.  Tried for salinity and wind speed too, but no luck.
